articles/data-factory/concepts-integration-runtime.md

```diff
@@ -58,6 +60,15 @@ An Azure integration runtime can:
 
 Azure Integration Runtime supports connecting to data stores and computes services with public accessible endpoints. Enabling Managed Virtual Network, Azure Integration Runtime supports connecting to data stores using private link service in private network environment.
 
+> [!NOTE]
+> In Synapse, workspaces have options to limit outbound traffic from the IR [managed virtual network](./synapse-analytics/security/synapse-workspace-managed-vnet)
+
+> [!NOTE]
+> In Data Factory, all ports are opened for [outbound communications](managed-virtual-network-private-endpoint.md#outbound-communications-through-public-endpoint-from-adf-managed-virtual-network).
+
+> [!NOTE]
```

Comment:
We aren't supposed to use adjacent notes. 3x seems too much.

Reply:
Ah, good point.  Should we merge these into a single note?

Reply:
In my opinion, it's not an "aside" callout anymore if the notes are longer than the main subject matter. Maybe regular paragraphs are OK too. I didn't read the whole page for full context.

Reply:
Ok good point!  I will rework it before publishing this one.